### DISCUSSION TITLE
Lotericas API Free

### DISCUSSION THREAD

**Discussion by rafaelferres** [BrasilAPI/BrasilAPI#230] [Ideas]
### Lotericas.io
Seguindo o mesmo motivo do projeto BrasilAPI, e também sou completamente contra pagar por ter dados públicos (todos os serviços de api que fornecem dados de lotericas são pagos WTF). Decidi criar a [Lotericas.io](https://lotericas.io/) que é uma API gratuita de resultados de loterias da caixa.

Acho que seria legal incluir essa api no BrasilAPI também.

### Tecnologias usadas 
- Next.js.
- Node.js e Puppeteer (captação dos dados no site da loterica).

### Gits
- Front: https://github.com/rafaelferres/lotericas-io
- Back: git em construção

**Comment by ItaloSa** [comment]
Acho que vale colocar um proxy pela BrasilApi pra chamar tua api

**Comment by brunofamiliar** [comment]
Tenho acompanhado a evolução desse projeto, e achei esse serviço extremamente útil!, concordo com o @ItaloSa
